src/main/java/taxi/service/PasswordUtil.java

```diff
@@ -0,0 +1,7 @@
+package taxi.service;
+
+public interface PasswordUtil {
+    String generateHash(String password);
+
+    boolean checkPassword(String password, String password1);
```

Comment:
```suggestion
    boolean checkPassword(String dbPassword , String loginPassword);
```

Reply:
![image](https://user-images.githubusercontent.com/74711003/230335594-f76a4f24-30c0-42d6-8b53-5a427ddaad11.png)

Strange, this should not have been in PR.